contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -194,7 +194,13 @@ contract VaultHub is PausableUntilWithRoles {
         // reserveRatio = TOTAL_BASIS_POINTS - maxMintableRatio
         // X = (mintedStETH * TOTAL_BASIS_POINTS - vault.valuation() * maxMintableRatio) / reserveRatio
 
-        return (mintedStETH * TOTAL_BASIS_POINTS - valuation * maxMintableRatio) / reserveRatioBP;
+        // Protection against underflow in division
+        if (!isHealthy && mintedStETH * TOTAL_BASIS_POINTS > valuation * maxMintableRatio) {
```

Comment:
There seems to be no reason to check for `!isHealthy` since there is a `isHealthy` check before that.

Reply:
agree